tests/conftest.py

```diff
@@ -100,10 +101,12 @@ def context(postgresql):
             'default': {
                 'backend': 'spinta.backends.postgresql:PostgreSQL',
                 'dsn': postgresql,
+                'dbName': 'spinta_test',
```

Comment:
Lets be consistent and use `camel_case` everywhere?

    dbName -> db_name

Or even better:

    dbName -> db

Reply:
Sorry, this comment was supposed to go for mongo, PostgreSQL does not need this configuration parameter at all. Each backend can have different configuration parameters.

Reply:
ok, will fix this.